modules/reserved-ips/README.md

```diff
@@ -0,0 +1,121 @@
+# IBM Reserved IP Module
+
+The module creates reserved IP's (https://cloud.ibm.com/docs/vpc?topic=vpc-managing-ip-addresses) on existing subnets. Reserved IP's can be assigned to your resources, for example VPE endpoint gateways.
+
+The module supports the following actions:
+- Create reserved IP addresses
+
+### Usage
+
+```hcl
+provider "ibm" {
+  ibmcloud_api_key = "XXXXXXXXXX" # pragma: allowlist secret
+  region           = "us-south"
+}
+
+# - Reserved IP
+module "ip" {
+  source           = "terraform-ibm-modules/vpe-gateway/ibm//modules/reserved-ips"
+  cloud_services = [
+    {
+      service_name = "kms"
+    },
+    {
+      service_name = "cloud-object-storage"
+    }
+  ]
+  subnet_zone_list = [
+    [
+      {
```

Comment:
Is there a reason for having to pass redundant data for the existing subnets that are targeted?

Reply:
`id`, `name`, and `zone` are used in the naming convention in [main](https://github.com/terraform-ibm-modules/terraform-ibm-vpe-gateway/blob/4eb433db29e2a59b06fc885a3fad54ef73c1b954/main.tf#L46) module so also needed [here](https://github.com/terraform-ibm-modules/terraform-ibm-vpe-gateway/pull/415/files#diff-9116576c0a8b977dbe986f7739366cfb48a4cfe0b1c3b7bf494929fd80965426R21) if names are to match.